wpilibc/src/main/native/include/frc/shuffleboard/SendableCameraWrapper.h

```diff
@@ -45,10 +59,20 @@ class SendableCameraWrapper : public Sendable,
    * @return a sendable wrapper object for the video source, usable in
    * Shuffleboard via ShuffleboardTab::Add() and ShuffleboardLayout::Add()
    */
-  static SendableCameraWrapper& Wrap(const cs::VideoSource& source);
-  static SendableCameraWrapper& Wrap(CS_Source source);
+  static SendableCameraWrapper& Wrap(const cs::VideoSource& source) {
+    return Wrap(source.GetHandle());
+  }
+  static SendableCameraWrapper& Wrap(CS_Source source) {
+    auto& wrapper = detail::GetSendableCameraWrapper(static_cast<int>(source));
+    if (!wrapper)
+      wrapper = std::make_shared<SendableCameraWrapper>(source, private_init{});
+    return *wrapper;
+  }
```

Comment:
If these wrap functions just took int’s, the cscore using parts could move to the ShuffleboardContainer functions.

Reply:
Wrap is technically allowed from user code, so I would want it to keep the actual types. But I realized I could do the same trick as ShuffleboardContainer to declare the functions as inline later.